app/models/concerns/revision_count.rb

```diff
@@ -0,0 +1,13 @@
+require 'active_support/concern'
+
+module RevisionCount
+  extend ActiveSupport::Concern
+
+  included do
+    after_update :increment_revision
+  end
+
+  def increment_revision
+    conference.increment!(:revision)
```

Comment:
Oh! The way it was before did not trigger the callbacks and that is why we didn't encounter the error at that time.
`update_attribute` or `update_column` will not trigger callbacks so we can maybe leave it the way it was before:
```ruby
conference.reload.increment(:revision)
conference.update_column(:revision, conference.revision)
```